mappings/net/minecraft/entity/ai/control/MoveControl.mapping

```diff
@@ -1,10 +1,12 @@
 CLASS net/minecraft/class_1335 net/minecraft/entity/ai/control/MoveControl
 	CLASS class_1336 State
 	FIELD field_6367 targetZ D
+	FIELD field_6368 forwardStrafe F
 	FIELD field_6369 targetY D
 	FIELD field_6370 targetX D
 	FIELD field_6371 entity Lnet/minecraft/class_1308;
 	FIELD field_6372 speed D
+	FIELD field_6373 sidewaysStrafe F
```

Comment:
This is redundant since "strafe" implies "sideways". Maybe either `strafe` or `sidewaysMovement`? I prefer `sidewaysMovement` to match `forwardMovement`.